trade-finance-manager-api/src/v1/helpers/create-tasks-amendment.helper.js

```diff
@@ -314,6 +316,27 @@ const updateAmendmentTasks = async (facilityId, amendmentId, taskUpdate) => {
   return originalTask;
 };
 
+const getTasksAssignedToUserByGroup = async (tasks, groupName, newUserId) => {
+  const updatedAt = getUnixTime(new Date());
+  const userFullName = await getAssigneeFullName(newUserId);
+
+  return tasks.map((group) => {
+    const assignedTasks = group.groupTasks.map((task) =>
+      task.team.id !== groupName
```

Comment:
task.team?.id - Optional chaining please.

Reply:
Done, but each task should have a team.